app/views/estimate_flow/applicant.html.slim

```diff
@@ -23,12 +23,26 @@
                                           hint: { text: t(".over_60.hint") }
 
     - if FeatureFlags.enabled?(:partner)
-      = form.govuk_collection_radio_buttons :partner,
-                                              yes_no_options,
-                                              :first,
-                                              :last,
-                                              legend: { text: t(".partner.legend") },
-                                              hint: { text: t(".partner.hint") }
+      = form.govuk_radio_buttons_fieldset(:partner, legend: { text: t(".partner.legend") }, hint: lambda { \
+                                                      tag.p(t(".partner.hint_text"), class: "govuk-hint") + \
+                                                      tag.ul(class: "govuk-list govuk-list--bullet") do \
+                                                        t(".partner.hint_list").map { |benefit| \
+                                                          tag.li(benefit, class: "govuk-hint") \
+                                                        }.join.html_safe \
```

Comment:
The linter hates this
'Avoid multi-line chains of blocks.'
Any suggestions on improving this are welcome.

Reply:
It's actually line 39 it's objecting to.